week-8/Homework/mandatory/1-practice/1-practice.md

```diff
@@ -24,8 +24,14 @@ The following endpoint is publicly available from Github
 
 1. Describe in a sentence what this API endpoint returns.
 
-<!-- Write your answer here -->
+<!--It returns a message which is not found(404) and object message
```

Comment:
Can you describe what it returns when something _was_ found? e.g. with the parameters you described in part 2?